explainer.md

```diff
@@ -286,6 +279,39 @@ typically associate with JS.
   * Large ArrayBuffers are not included in usedJSHeapSize and totalJSHeapSize,
     since they are backed by PartitionAlloc.
   * External strings are not included in usedJSHeapSize and totalJSHeapSize
-  
+* The numbers are quantized into buckets and time-delayed.
+
 Moreover performance.memory in Chrome exposes jsHeapSizeLimit which is an JavaScript engine specific
-heuristic which may not exisit in other JavaScript engines. Therefore, we propose to remove this metric.
+heuristic which may not exisit in other JavaScript engines.
+
+# <a name="appendix-d"></a> Appendix D - Frame Tree Examples
+
+Note: All examples assume site isolation - frames from different origins are
```

Comment:
Should we remove the term site isolation here and only say "Note: All examples assume that frames..."

Reply:
Done.